### ISSUE TITLE
uikit 3 : uk-switcher not working with uk-dotnav and uk-slidenav

### ISSUE THREAD

**Issue by trk** [uikit/uikit#2313]
I am trying to make a quick slideshow with [uk-swither](https://getuikit.com/docs/switcher) + [uk-dotnav](https://getuikit.com/docs/dotnav) + [uk-slidenav](https://getuikit.com/docs/slidenav), but in my tests uk-switcher not working with these components.

My code:
```html
<div class="uk-card uk-card-default">
    <div class="uk-card-media-top">
        <div class="uk-inline uk-visible-toggle uk-light">
            <ul class="uk-switcher slide-container uk-margin">
                <li class='uk-active'>
                    <img src="image1.jpg" alt="">
                    <!-- Not Working -->
                    <a class="uk-position-center-left uk-position-small uk-hidden-hover uk-slidenav-large" uk-slidenav="previous" data-uk-switcher-item="previous"></a>
                    <a class="uk-position-center-right uk-position-small uk-hidden-hover uk-slidenav-large" uk-slidenav="next" data-uk-switcher-item="next"></a>
                </li>
                <li>
                    <img src="image1.jpg" alt="">
                    <!-- Not Working -->
                    <a class="uk-position-center-left uk-position-small uk-hidden-hover uk-slidenav-large" uk-slidenav="previous" data-uk-switcher-item="previous"></a>
                    <a class="uk-position-center-right uk-position-small uk-hidden-hover uk-slidenav-large" uk-slidenav="next" data-uk-switcher-item="next"></a>
                </li>
            </ul>
            <!-- Not Working -->
            <div class="uk-position-bottom-center uk-position-medium">
                <ul class="uk-dotnav uk-flex-nowrap" data-uk-swither="connect: .slide-container">
                    <li class="uk-active"><a>Item 1</a></li>
                    <li><a>Item 2</a></li>
                </ul>
            </div>
        </div>
    </div>
    <div class="uk-card-body">
        <!-- Standard usage from uikit-doc -->
        <ul class="uk-subnav uk-subnav-pill" data-uk-switcher="connect: .slide-container">
            <li class="uk-active"><a>Item 1</a></li>
            <li><a>Item 2</a></li>
        </ul>
    </div>
</div>
```

**Comment by trk** [comment]
On my latest tests, uk-switcher working with uk-dotnav and uk-slidenav problem cause from uk-swircher-item for uk-slidenav.

**data-uk-switcher-item not working !**, if i remove **data- prefix** from uk-switcher-item next prev options working.
```html
<div class="uk-card uk-card-default">
    <div class="uk-card-media-top">
        <div class="uk-position-relative">
            <ul class="uk-switcher slide-container uk-margin">
                <li class='uk-active'>
                    <img src="image1.jpg" alt="">
                    <div class="uk-slidenav-container uk-position-top-left uk-position-small">
                        <a class="uk-slidenav-large" uk-slidenav="previous" uk-switcher-item="previous"></a>
                        <a class="uk-slidenav-large" uk-slidenav="next" uk-switcher-item="next"></a>
                    </div>
                </li>
                <li>
                    <img src="image2.jpg" alt="">
                    <div class="uk-slidenav-container uk-position-top-left uk-position-small">
                        <a class="uk-slidenav-large" uk-slidenav="previous" uk-switcher-item="previous"></a>
                        <a class="uk-slidenav-large" uk-slidenav="next" uk-switcher-item="next"></a>
                    </div>
                </li>
            </ul>
            <div class="uk-position-bottom-center uk-position-medium">
                <ul class="uk-dotnav uk-flex-nowrap" data-uk-switcher="connect: .slide-container">
                    <li class='uk-active'><a>Slide 0</a></li>
                    <li><a>Slide 1</a></li>
                </ul>
            </div>
        </div>
    </div>
    <div class="uk-card-body">
        <ul class="uk-subnav uk-subnav-pill" data-uk-switcher="connect: .slide-container">
            <li class="uk-active"><a>Item 1</a></li>
            <li><a>Item 2</a></li>
        </ul>
    </div>
</div>
```

**Comment by janschoenherr** [comment]
Fixed with d0aae7dfb553843be3cb94002ba51aa2ef0aef0f

If the problem persists, please reopen.
